ImageCone[] cones
uint16 imageWidth
uint16 imageHeight
time timestamp
